stdlib/3.4/enum.pyi

```diff
@@ -20,8 +21,6 @@ class Enum:
 class IntEnum(int, Enum):
     value = ...  # type: int
 
-_T = TypeVar('_T')
-
 def unique(enumeration: _T) -> _T: ...
```

Comment:
Hm...
```
import enum
class Colors(enum.Enum):
  RED = 1
  GREEN = 2
  BLUE= 3
X = unique(Colors)
reveal_type(X)
```
Gives
```
__tmp__.py:12: error: Type argument 1 of "unique" has incompatible value "Colors"
__tmp__.py:13: error: Revealed type is 'def (value: Any) -> __tmp__.Colors'
```
(Sorry the line numbers are offset, it's about the last two lines.)